src/content/ui-api/web/app-platform/index.md

```diff
@@ -3,7 +3,7 @@ linkTitle: App platform
 title: The Giant Swarm App Platform in the web interface
 description: What the Giant Swarm App Platform looks like on our web interface and how to use it.
```

Comment:
I'm confused about our usage of App Platform and App Catalog/Managed Apps. Isn't App Platform a set of capabilities that enables App Catalog/Managed Apps?
If so, I recommend changing the wording. Ping @cokiengchiara / @rossf7

Reply:
That's not part of this change.

I would update last_review_date if I actually went over these entire documents. This PR here is about adding pages.

Reply:
@marians Yes this PR isn't changing the text but @Oshratn I think it's correct as is.

> In short: the Giant Swarm App Platform refers to the whole feature, and an App Catalog is a collection of Apps.

This is from https://docs.giantswarm.io/app-platform/overview/ and there is more explanation there.

Reply:
Unblocking the PR. 

@rossf7 can we work on separate PR to clarify the language around App Platform and App Catalog?